utils/travis/common.sh

```diff
@@ -5,7 +5,7 @@ if command -v nproc >/dev/null; then
 	export CPUS_USED=$(nproc)
 fi
 
-add_apt_repos() {
+add_qt_apt_repos() {
```

Comment:
that's not only for qt

Reply:
How should I call it then? only add_apt_repos is not an option.

Reply:
Can we maybe use `add_${PLATFORM}_apt_repos`?